requirements.txt

```diff
@@ -2,4 +2,5 @@ setuptools~=47.3.1
 lxml~=4.5.0
 pytz~=2020.1
 pytest-cov
-mysql-connector
\ No newline at end of file
+mysql-connector
```

Comment:
I would suggest setting pytest-cov and mysql-connector versions. If latest works (which I assume it does since that is whats being installed) then just set them to that version.

This will prevent users from breaking in the future. Also dependabot to report to you when new versions appear and run through your tests to make sure its compatible.